m4/pbs_default_scheduler.m4

```diff
@@ -37,12 +37,7 @@
 
 AC_DEFUN([PBS_AC_DEFAULT_SCHEDULER],
 [
-  AC_MSG_CHECKING([for Default scheduler name])
```

Comment:
It depends on whether you want to override it at compile time. If that were the case, then this macro should be renamed with_default_scheduler.m4 and the content be similar to with_tmpdir.m4. As the code currently stands, it could just as easily be a #define in another header file.